example/src/example/__tests__/FbteeStrings-test.tsx

```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { test } from '@jest/globals';
+import { CollectFbtOutput } from '../../../../packages/babel-plugin-fbtee/src/bin/collect.tsx';
+
+test('fbtee strings are included in the collected strings of the example project', () => {
```

Comment:
This test verifies that the strings from `fbtee` are correctly included in the example project via the `babel-plugin-fbtee` collection script.